### ISSUE TITLE
TextBox value={undefined} should leave input empty

### ISSUE THREAD

**Issue by davecaruso** [Tropix126/fluent-svelte#7]
### Before you start...

- [X] Have you updated your dependencies? You might be using an old version of the library.
- [X] Have you checked for an existing issue on this topic? If there is one, post a comment on it instead.

### What browsers are you seeing the problem on?

Microsoft Edge

### Description

i use this pattern a lot
```svelte
<input bind:value={object.property} />
```
and if the value is unset or otherwise undefined, the input shows `undefined`. the way the TextBox is implemented, it just uses the value prop, not bind, which doesnt copy this behavior over, using TextBox or <input> without the `bind:`, it will put a literal "undefined" in the text box.

### Steps To Reproduce

```svelte
<script>
  const object = { };
</script>

<TextBox type="text" bind:value={object.property} />

<code>{JSON.stringify(object)}</code>
```

### Expected behavior

the input has no text in it to start

**Comment by davecaruso** [comment]
this can be fixed with a one line change to the textbox component, changing `{value}` to `value={value ?? ""}`

**Comment by Tropix126** [comment]
This seems to have been fixed as of 1.1.0, honestly don't know exactly *what* fixed it though.
